content/handbook/support/workflows/data_processing_addendums.md

```
@@ -0,0 +1,35 @@
+stale: false
+---
+
+## 概要
+
+データ処理付帯条項は、Sales 担当者を通じて GitLab.com の有料サブスクリプション加入者が利用できます。このワークフローでは、顧客から受け取った完全に締結 (署名) された DPA をどう処理するかを詳述します。
+
+## ワークフロー
+
+>**注:** このワークフローは、ユーザーが GitLab.com の **有料サブスクリプションを保有している** 場合にのみ適用されます。
+
+1. 顧客の Salesforce アカウントを見つけます。正しい文字列は `001` で始まり、Zendesk の組織[1] の下、Zendesk チケットに自動的に投稿されたメモにリンクされている[2]、または CustomersDot[3] の **Edit** インターフェースから見つけることができます ([スクリーンショット](#screenshots) は下記)。Salesforce アカウントレコードの URL は次のようになります: `https://gitlab.my.salesforce.com/0014M00001hHV5CQAW`。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)